frontend/components/forms/fields/InputField/InputField.jsx

```diff
@@ -36,6 +37,7 @@ class InputField extends Component {
     onFocus: noop,
     type: "text",
     value: "",
+    target: false,
```

Comment:
Can we change this prop name to something that answers true/false? Like `includeTarget` or `parseTarget`.